src/coreclr/jit/gentree.h

```diff
@@ -5221,6 +5218,15 @@ struct GenTreeCall final : public GenTree
         return (gtFlags & GTF_CALL_INLINE_CANDIDATE) != 0;
     }
 
+#ifdef SWIFT_SUPPORT
+    bool HasSwiftErrorHandling()
+    {
+        // Most calls aren't Swift calls, so short-circuit this check by checking the calling convention first.
+        return (unmgdCallConv == CorInfoCallConvExtension::Swift) &&
```

Comment:
Accessing `unmgdCallConv` here is UB when that's not the active union member, so this probably needs a few other checks.